python/ray/util/client/dataclient.py

```diff
@@ -19,6 +21,9 @@
 
 ResponseCallable = Callable[[ray_client_pb2.DataResponse], None]
 
+# How often to make manual KeepAlive calls
+DATACLIENT_KEEPALIVE_INTERVAL = 30
```

Comment:
should we make it less than the product's interval which IIRC is set to 15?
@ijrsvt knows more, most likely I am wrong here.

Reply:
Set to 10 seconds for now just to be safe.